packages/cli/src/index.js

```diff
@@ -13,7 +13,8 @@ program
 program
   .command("preview")
   .description("Set up your emails directory structure.")
-  .action(() => require("./commands/preview")());
+  .option('-s, --source <path>', 'The source of your templates', 'emails')
```

Comment:
Nice! Do we necessarily need to use a flag for it, or could we make it an optional argument? That might be cleaner, and then the name doesn't matter :)

```
macaw preview ./emails
```

Reply:
Not necessarily but the rational for making it an option rather than 3rd argument was guided by the fallback behaviour.

Most cli tools I have used fallback to sensible defaults if an option is not passed but error if an argument is not supplied.

Git for example errors when you don't provide a glob

```bash
git add
Nothing specified, nothing added.
Maybe you wanted to say 'git add .'?
```

But git commit falls back to vim if you don't provide the -m command.

So the idea is that with that

```bash
macaw preview
```

will load from a folder called emails

but --source templates will load from the templates
```bash
macaw preview --source templates
```

and potentially even could do things in the future like this
```bash
macaw preview --source http://s3.amazonaws.com/bucket/ --source-loader s3
```

Reply:
Nice! That way of thinking about it makes total sense!